transcript/.gitignore

```diff
@@ -31,3 +31,5 @@ build/
 
 ### VS Code ###
 .vscode/
+
+apiAccess.properties
```

Comment:
Add a note about apiAccess.properties for collaborators so that they know to add the file manually to their code.